### ISSUE TITLE
Installing component via Vue.use causes error in old browsers

### ISSUE THREAD

**Issue by gamtiq** [mlinquan/vue-awesome-countdown#3]
If I install component via `Vue.use` and don't add it to `transpileDependencies` option of Vue CLI, I get the following error on browsers that don't support [Enhanced Object Literals](https://github.com/lukehoban/es6features#enhanced-object-literals):
> Uncaught SyntaxError: Unexpected token (

The error arises because of the following code in [vue-awesome-countdown.js](https://github.com/mlinquan/vue-awesome-countdown/blob/master/src/vue-awesome-countdown.js):
```js
  install(Vue, name) {
    name = name || 'vac'
    Vue.component(name, vueAwesomeCountdown)
    Vue.component('countdown', vueAwesomeCountdown)
  }
```
To fix the issue it is necessary change [line #4](https://github.com/mlinquan/vue-awesome-countdown/blob/master/src/vue-awesome-countdown.js#L4) to:
```js
install: function install(Vue, options) {
```
To work around the issue, install components by calling `Vue.component`:
```js
import vueAwesomeCountdown from 'vue-awesome-countdown/vue-awesome-countdown.vue';
...
Vue.component('vac', vueAwesomeCountdown);
```

**Comment by mlinquan** [comment]
Which version do you use?

**Comment by mlinquan** [comment]
> If I install component via `Vue.use` and don't add it to `transpileDependencies` option of Vue CLI, I get the following error on browsers that don't support [Enhanced Object Literals](https://github.com/lukehoban/es6features#enhanced-object-literals):
> 
> > Uncaught SyntaxError: Unexpected token (
> 
> The error arises because of the following code in [vue-awesome-countdown.js](https://github.com/mlinquan/vue-awesome-countdown/blob/master/src/vue-awesome-countdown.js):
> 
> ```js
>   install(Vue, name) {
>     name = name || 'vac'
>     Vue.component(name, vueAwesomeCountdown)
>     Vue.component('countdown', vueAwesomeCountdown)
>   }
> ```
> 
> To fix the issue it is necessary change [line #4](https://github.com/mlinquan/vue-awesome-countdown/blob/master/src/vue-awesome-countdown.js#L4) to:
> 
> ```js
> install: function install(Vue, options) {
> ```
> 
> To work around the issue, install components by calling `Vue.component`:
> 
> ```js
> import vueAwesomeCountdown from 'vue-awesome-countdown/vue-awesome-countdown.vue';
> ...
> Vue.component('vac', vueAwesomeCountdown);
> ```

Thank you your feedback, Repaired in Version 1.0.12.
